vizro-core/src/vizro/_constants.py

```diff
@@ -8,3 +8,4 @@
 FILTER_ACTION_PREFIX = "filter_action"
 PARAMETER_ACTION_PREFIX = "parameter_action"
 ACCORDION_DEFAULT_TITLE = "SELECT PAGE"
+MIN_NO_OF_PAGES = 2
```

Comment:
I don't think this is a constant we would ever like to change. It will always be in case of one page. I assume we have 2 here, because of the 404 error page? 

I would just hard-code that in the code and add a comment there, because this constant is not supposed to be changed.

Reply:
Since we will treat each icon/navitem individually and agreed to remove the nav_controls_panel if there is one page and no controls then this check if no longer needed, and I will remove both the check and the constant.